src/utils/archive.ts

```diff
@@ -80,6 +81,16 @@ const Archive = {
 
     Editor.edits.apply ( doc.textEditor, edits );
 
+    var previousParsedDate;
+
+    function extractDate (line: string) {
+      let result = line.match ( Consts.regexes.tagFinished );
+      if (!result) return previousParsedDate;
```

Comment:
It doesn't make sense to return the previous finished date in this case, it should return `- Inifinity` or something, maybe just `0` or `-1`, so that those lines will be then sorted by line number.

Reply:
I remember I added it for comments, so comment line will take the same date as parent task